### PR TITLE
Exclude lpsolve from python interface

### PR DESCRIPTION
**Pull request by TolisChal**
-- This PR adds a macro `volestipy`. When it is defined the lpsolve is not included.  
-- The python interface does not use the library `lpsolve`.
-- Change the typedef declarations in `\external/minimum_ellipsoid/khach.h` to fix the compiling of python interface

### CODE REVIEW

File: volestipy/volestipy/include/bindings.h [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
+#define VOLESTIPY
+#include <cmath>
 // from SOB volume - exactly the same for CG and CB methods
 #include <fstream>
 #include <iostream>
 #include "random_walks.hpp"
 #include "random.hpp"
-#include "misc.h"
-#include "known_polytope_generators.h"
+//#include "misc.h"
```
Comment (vissarion) [inline volestipy/volestipy/include/bindings.h]: should we just remove them?

File: include/random_walks/gaussian_cdhr_walk.hpp [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #define RANDOM_WALKS_GAUSSIAN_CDHR_WALK_HPP
 
 #include "generators/boost_random_number_generator.hpp"
+#include "sampling/sphere.hpp"
```
Comment (vissarion) [inline include/random_walks/gaussian_cdhr_walk.hpp:11]: this should go below `#include "random_walks/gaussian_helpers.hpp"` ?

**Review by vissarion — COMMENTED** [review]
First, thanks for the PR.
The commit is not very clear, it contains the additions for the macro but also some unrelated changes, like this https://github.com/GeomScale/volume_approximation/pull/135/files#diff-01f6b859ad40db2a9ab913e17f5b612b1efbbd7aab1bd73f981289a70d11571aR39
I think it could be clearer to split the work in two commits, one for the VOLESTIPY macro and one for the rest.

File: include/random_walks/gaussian_cdhr_walk.hpp (hunk 2 above)
Comment (TolisChal) [reply include/random_walks/gaussian_cdhr_walk.hpp:11]: Yes it should be above `#include "random_walks/gaussian_helpers.hpp"`. Done

File: volestipy/volestipy/include/bindings.h (hunk 1 above)
Comment (TolisChal) [reply volestipy/volestipy/include/bindings.h]: Yes of course. Done

**Comment by TolisChal** [comment]
The changes in https://github.com/GeomScale/volume_approximation/pull/135/files#diff-01f6b859ad40db2a9ab913e17f5b612b1efbbd7aab1bd73f981289a70d11571aR39 fix the compile of python interface

**Review by vissarion — APPROVED** [review]

**Pull request merged**